purchase_cancel_reason/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,7 @@
+* Guewen Baconnier, Camptocamp SA
+* Sylvain Van Hoof <sylvain@okia.be>
+
+* `Ecosoft <http://ecosoft.co.th>`_:
+
+  * Kitti U. <kittiu@ecosoft.co.th>
+  * Tharathip C. <tharathipc@ecosoft.co.th>
```

Comment:
reformat list: remove blank lines and remove indent

Reply:
@jbaudoux This shoul remains as is as they are regrouped into the same company.

Looks like:

![image](https://user-images.githubusercontent.com/19529533/231391858-b3c9976c-9ba5-4f7b-b61d-1ee3eb2a9426.png)

Reply:
I reverted this commit so it looks like it used to